### DISCUSSION TITLE
请问 这套主题是要vue启动吗，有什么需要注意的吗

### DISCUSSION THREAD

**Discussion by mobeicanyue** [zdz/ServerStatus-Rust#189] [Q&A]
小白，已经部署好了客户端和服务端，现在想换主题，但是研究发现好像要用VUE
用VUE部署有什么要求吗
![image](https://user-images.githubusercontent.com/81098819/233593169-1e0f1d3a-ecaf-4d29-8ecf-d6821513083b.png)

**Comment by zdz** [comment]
不懂，去主题那边问问
